ml-agents/mlagents/trainers/tests/test_simple_rl.py

```diff
@@ -204,7 +205,7 @@ def _check_environment_trains(env, config):
         seed = 1337
 
         trainer_config = yaml.safe_load(config)
-        env_manager = SimpleEnvManager(env)
+        env_manager = SimpleEnvManager(env, FloatPropertiesChannel())
```

Comment:
should this be a list, e.g. `[FloatPropertiesChannel()]`?

Reply:
Never mind, got my Environments and EnvironmentManagers mixed up.

Reply:
I made the constructor of SimpleEnvManager take only a FlotPropertiesChannel as input so we can manage the reset_parameters. SimpleEnvManager does not have generic sideChannels